client/components/mobile/IconButton.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from 'styled-components';
+import { prop, remSize } from '../../theme';
+
+const textColor = prop('primaryTextColor');
+
+const IconButton = styled.button`
+width: 3rem;
```

Comment:
Would it make sense to extend from `client/common/Button`? No worries if that's overcomplicated. Either way, I'd like this to use `remSize` so the measurement can easily be read as pixels.

Reply:
Sure! I updated it on